### ISSUE TITLE
How can I add a Period to a Date?

### ISSUE THREAD

**Issue by pmullot** [CaptorAB/quantlib-wasm#9]
I'm trying to add a Period to a Date. 
This is the code I use.
```
const { Date: QlDate, Period, TimeUnit } = QuantLib;
const date = new QlDate.fromISOString("2023-08-19");
 const p = new Period(4, TimeUnit.Months);
const in4m = today + p; // this yields 'August 19th, 20234M'
```

As you can see, you can't use "+" to add the 2 objects since the library just concat the 2 string representations of the objects.
What is the proper way to do that?

**Comment by oddball** [comment]
@AndersModin can you perhaps help?

**Comment by AndersModin** [comment]
The code should work the way you describe. This is the way to handle QuantLib dates in c++ or Python. Unfortunately it seems that class operators is not available via emscripten. At least not as `quantlib-embind.cpp` is set up today. Thus you get this strange behaviour.

One workaround would be to use the `calendar` class, basically any calendar would do since we will use `Unadjusted` and `advance`

so basically

``` javascript
const { Date: QlDate, TimeUnit, BusinessDayConvention, Sweden } = QuantLib;
const date = new QlDate.fromISOString("2023-08-19");
const cal = new Sweden();
const in4m =cal.advance(date, 4, TimeUnit.Months, BusinessDayConvention.Unadjusted, false)
```

would do the trick.

**Comment by pmullot** [comment]
Awesome! 
I'll try it out!

Thanks a lot!
